### PR TITLE
Simplify fix for abrupt panel transition

### LINKED ISSUE #2298: Simplify fix for panel abruptly transitioning due to bottom panel button
### Please confirm that you have searched existing issues in the repo

Yes, I have searched the existing issues

### Any related issues?

#2200

### What is the area that this feature belongs to?

_No response_

### Is your feature request related to a problem? Please describe.

https://github.com/MarkBind/markbind/pull/2200/files#r1177849315

### Describe the solution you'd like

See the linked comment

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by luminousleek**
**What is the purpose of this pull request?**

- [ ] Documentation update
- [ ] Bug fix
- [ ] Feature addition or enhancement
- [x] Code maintenance
- [ ] DevOps
- [ ] Improve developer experience
- [ ] Others, please explain:

Fixes #2298

**Overview of changes:**
* Wrap the collapse button at the bottom of a panel in a `<div>` block and add bottom padding it. 
* Revert the changes in #2200.

**Anything you'd like to highlight/discuss:**
`padding-bottom` and `margin-bottom` are needed to prevent margin collapse 

**Testing instructions:**

NA

**Proposed commit message: (wrap lines at 72 characters)**

```
Simplify fix for abrupt panel transition. 

The previous fix implemented a method to compute the max
height of a panel by checking if the collapse button is present
and adding the height of the bottom margin of the button.

Let's simplify this by wrapping the button in a <div> block and 
adding padding to the block. Let's also remove the getMaxHeight
method and its associated tests.
```

---
**Checklist:** :ballot_box_with_check:

- [ ] Updated the documentation for feature additions and enhancements
- [ ] Added tests for bug fixes or features
- [x] Linked all related issues
- [x] No unrelated changes 

---

**Reviewer checklist:**

Indicate the [SEMVER](https://semver.org/) impact of the PR:
- [ ] Major (when you make incompatible API changes)
- [x] Minor (when you add functionality in a backward compatible manner)
- [ ] Patch (when you make backward compatible bug fixes)

At the end of the review, please label the PR with the appropriate label: `r.Major`, `r.Minor`, `r.Patch`.

Breaking change release note preparation (if applicable):
- To be included in the release note for any feature that is made obsolete/breaking

> Give a brief explanation note about:
>   - what was the old feature that was made obsolete
>   - any replacement feature (if any), and
>   - how the author should modify his website to migrate from the old feature to the replacement feature (if possible).

### CODE REVIEW

File: packages/vue-components/src/panels/NestedPanel.vue [hunk 1]
```diff
@@ -92,11 +92,13 @@
             :fragment="fragment"
             @src-loaded="retrieverUpdateMaxHeight"
           />
-          <panel-switch
-            v-show="isExpandableCard && bottomSwitchBool"
-            :is-open="localExpanded"
-            @click.native.stop.prevent="toggle(true)"
-          />
+          <div class="card-bottom">
```
Comment (yucheng11122017) [inline packages/vue-components/src/panels/NestedPanel.vue:95]: Maybe `bottom-button-wrapper` is a better class name? Its similar to what we call our button-wrapper class now :)

**Review by yucheng11122017 — COMMENTED** [review]
Hi @luminousleek thanks so much for the work! The solution seems simple and effective. Kind of kicking myself for not doing this in the first place. 
One minor nit on the class name and we should be good to do!

**Review by yucheng11122017 — DISMISSED** [review]
LGTM! Thanks for the work @luminousleek :))

**Comment by yucheng11122017** [comment]
Hi @luminousleek could you run `npm updatetest` to update the expected test cases :)

File: packages/cli/test/functional/test_site_algolia_plugin/expected/index.html [hunk 2]
```diff
@@ -17,7 +17,6 @@
   <script src="/test_site_algolia_plugin/markbind/js/markbind.min.js"></script>
   <script src="index.page-vue-render.js"></script>
   <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@docsearch/css@3.2.0/dist/style.css">
-  <link rel="preconnect" href="https://R2IYF7ETH7-dsn.algolia.net" crossorigin>
```
[the remark this answers is not in the document]
Comment (luminousleek) [reply packages/cli/test/functional/test_site_algolia_plugin/expected/index.html]: hmm looks like the deleted line was from the previous PR. the lines added in the `algolia.ts` file in [here](https://github.com/MarkBind/markbind/pull/2406/files) are there too. anyhow i've manually added this line back.

**Review by yucheng11122017 — APPROVED** [review]
LGTM! Thanks a bunch @luminousleek  :)

**Pull request merged**